postinstall.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+/* eslint-disable no-console, promise/prefer-await-to-then */
+
+async function main () {
+  const driverEnv = process.env.npm_config_drivers;
+  const pluginEnv = process.env.npm_config_plugins;
+
+  if (!driverEnv && !pluginEnv) {
+    console.log('Not auto-installing any drivers or plugins');
+    return;
+  }
+
+  let extension;
+  try {
+    extension = require('./build/lib/cli/extension');
+  } catch (e) {
+    throw new Error(`Could not load extension CLI file; has the project been transpiled? (${e})`);
+  }
+
+  const {DEFAULT_APPIUM_HOME, DRIVER_TYPE, PLUGIN_TYPE} = require('./build/lib/extension-config');
+  const {runExtensionCommand} = extension;
+  const appiumHome = process.env.npm_config_appium_home || DEFAULT_APPIUM_HOME;
+  const specs = [[DRIVER_TYPE, driverEnv], [PLUGIN_TYPE, pluginEnv]];
+
+  for (const [type, extEnv] of specs) {
+    if (extEnv) {
+      for (const ext of extEnv.split(',')) {
+        const extList = await runExtensionCommand({
```

Comment:
I assume we should be more flexible here and skip possible failing exensions + log errors rather than terminating the whole postinstall process

Reply:
hmmm, yeah good point